functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,15 @@
  */
 struct BeerOrganizer
 {
+private:
```

Comment:
Please never commit previous tasks in the new one!
Start each task from the master branch